### PR TITLE
Fix bug for picking last used channel, when all channels are empty

### PR DESCRIPTION
**Pull request by AleshaOleg**

### CODE REVIEW

File: src/utils/backend/WampClient.js [hunk 1]
```diff
@@ -136,6 +135,7 @@ export default class WampClient {
   }
 
   onDisconnected = () => {
+    this.backoff.reset()
```
Comment (nikgraf) [inline src/utils/backend/WampClient.js]: Why did you move the backoff reset? How is it related to the channel change?

Comment (AleshaOleg) [reply src/utils/backend/WampClient.js]: When I try to reopen websocket connection, setTimeout don't worked properly, because on every reopen we have `backoff.reset()`. Here we call `this.open()` - https://github.com/ubergrape/grape-web-client/blob/master/src/utils/backend/WampClient.js#L73 and then it's trigger - https://github.com/ubergrape/grape-web-client/blob/master/src/utils/backend/WampClient.js#L121

I'm deleted that, in order to on each reopen setTimeout work properly (like try to reconnect each 100, 200, 400, 800, 1600ms). Previously it's tried to reconnect each 100ms  because on each `reopen` backoff reset to initial value.

Comment (AleshaOleg) [reply src/utils/backend/WampClient.js]: It's related to this issue, because if something went wrong on initial data load, user will see reconnectiong each 100ms (disco-mode enabled:D)

Comment (nikgraf) [reply src/utils/backend/WampClient.js]: Thanks for the detailed explanation!
